### ISSUE TITLE
Animations and Tilemap Editor broken on iPad when using pen and touch in "add to home" version

### ISSUE THREAD

**Issue by Freshtefan** [Scirra/Construct-3-bugs#4891]
## Problem description

Anytime I change from drawing with the pen to zooming with fingers the editor defaults to zooming tool and won’t change back no matter what tool I click. It reverts to normal use when closed and opened back up, but every time I change input method it bugs out again.

## Attach a .c3p

no c3p attached, it’s reproducible in every project 

## Steps to reproduce

use IPad Pro 12 inch from 2018
Open construct 3 from the Home Screen 
Edit a sprite or tilemap 
Draw with the pen
Zoom with fingers

## Observed result

using fingers or pen only results in zooming on or out slowly

## Expected result

zooming when pinching fingers, drawing when using pen

## More details

it does not seem to happen in a safari tab

**Affected browsers/platforms:** 

**First affected release:** 

## System details

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by DiegoScirra** [comment]
Does this work properly if you only use fingers to both zoom and draw?

**Comment by Freshtefan** [comment]
Yes, sticking to either input method works, but mixing pen and finger is trouble
